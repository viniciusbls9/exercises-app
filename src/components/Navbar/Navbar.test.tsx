import { describe, it, expect } from 'vitest';
import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom';

import Menu from './';

describe('Menu', () => {
  it('renders logo and links', () => {
    const { container } = render(<Menu />);

    const logo = screen.getByAltText('Logo');
    expect(logo).toBeInTheDocument();

    // // Check if the links are rendered
    const homeLink = screen.getByText('Home');
    expect(homeLink).toBeInTheDocument();

    const exercisesLink = screen.getByText('Exercises');
    expect(exercisesLink).toBeInTheDocument();
    expect(container).toMatchSnapshot()
  });
});
