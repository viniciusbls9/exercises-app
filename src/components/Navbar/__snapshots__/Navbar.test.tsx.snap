// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Menu > renders logo and links 1`] = `
<div>
  <div
    class="sc-aXZVg eULosh"
  >
    <div
      class="sc-gEvEer gpdSm"
    >
      <img
        alt="Logo"
        class="sc-eqUAAy fqZEnF"
        src="/logo.png"
      />
      <div
        class="sc-fqkvVR dKFVrM"
      >
        <a
          class="sc-dcJsrY kERCCE"
          href="#"
        >
          Home
        </a>
        <a
          class="sc-dcJsrY kERCCE"
          href="#"
        >
          Exercises
        </a>
      </div>
    </div>
  </div>
</div>
`;
